code/stable/hghc/SRS/PDF/HGHC_SRS.tex

```diff
@@ -115,6 +115,7 @@ \subsubsection{Data Definitions}
 \\ \bottomrule
 \end{tabular}
 \end{minipage}
+
 \vspace{\baselineskip}
```

Comment:
I'm additionally wondering if we should put some wiggle room (aka `glue` in TeX parlance)?